lib/move/__tests__/cross-device-utils.js

```diff
@@ -0,0 +1,21 @@
+const fs = require('graceful-fs')
+const path = require('path')
+
+// USER CONFIGURATION
+let runCrossDeviceTests = false // change to true to enable
```

Comment:
Wouldn't it make sense to use environment variables for these two settings? This would prevent commiting changes by accident.

A benefit would be that we could document that in the README and make it visible (with the downside of maintaining the README of these env variables).

Reply:
Good idea; could actually unify to a single environment variable for the path; if it's set, we run the tests, if it's not set, we don't. Will refactor this when I get a chance.